actfw_core/unicam_isp_capture.py

```diff
@@ -186,6 +189,11 @@ def __init__(
 
         self.request_buffer()
 
+    def get_sensor_name(self) -> str:
+        with open("/sys/class/video4linux/v4l-subdev0/device/name") as f:
```

Comment:
Is it ok to have `v4l-subdev0` hard-coded? (for example, when multiple cameras are connected)